src/routes/index.tsx

```diff
@@ -89,7 +97,7 @@ export const PAGES = [
 
 const getRoutesList = () => {
   const pages = PAGES
-  const RedirectToDashboard = () => <Navigate to={pages[0].path} replace />
+  const RedirectToHome = () => <Navigate to={pages[0].path} replace />
```

Comment:
let's do this change after the page is ready

Reply:
good idea